src/handlers/testing/exec/util/run-manager.ts

```diff
@@ -140,6 +140,10 @@ export class RunManager {
         isDefaultBranch: ciContext.isDefaultBranch,
         ciProvider: ciContext.ciProvider,
         buildHtmlUrl: ciContext.buildHtmlUrl,
+        workflowId: ciContext.workflowId,
+        workflowName: ciContext.workflowName,
+        workflowRunNumber: ciContext.workflowRunNumber,
+        jobName: ciContext.jobName,
```

Comment:
this will allow us to distinguish reruns in the UI (rn they look like duplicates)